### ISSUE TITLE
Integration not prompting me to enter host....

### ISSUE THREAD

**Issue by BluThunder2k** [siku2/hass-weatherlink#20]
I've install the integration, and it never prompts me to enter a host, nor can I find anywhere to set it...

Suggestions?

**Comment by siku2** [comment]
Did you only install it using HACS or did you also try to 'add' it in the Home Assistant integrations menu?

**Comment by BluThunder2k** [comment]
I did it thru the integrations with the GitHub auth code and selecting it.    Is there another way?   I'm fairly new to home assistant.

**Comment by siku2** [comment]
That doesn't really tell me what you're doing.
There are two distinct steps involved:

1. You need to install the integration using HACS. This only downloads the integration to your home assistant, it doesn't automatically set it up.
2. You need to add the integration to the home assistant. Now that the integration is installed this can be done the same way built-in integrations are added; through the integrations config menu. This is where you'll be prompted for the weatherlink host.

Which part are you struggling with? If you're following a tutorial it would be helpful to know which one.

**Comment by BluThunder2k** [comment]
Ah, that's where I went off the track.   I was following a suggestion that in home assistant you add a integration called github, authorize it, and then if you star'd your repository, it came up as a option to install.   Which it does, but it never lets you configure it.    I did it thru HACS this time and it worked perfectly.   Thanks for the nudge in the right direction, even though I should have read the page better.
